tests/test_toolsqa_alerts.py

```diff
@@ -0,0 +1,168 @@
+import time
+import unittest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import UnexpectedAlertPresentException, TimeoutException, NoAlertPresentException, \
+    NoSuchElementException
+from utils.base_test import BaseTest
+
+BASE_URL = 'https://demoqa.com/alerts'
+INSTANT_ALERT_BUTTON_ID = 'alertButton'
+DELAYED_ALERT_BUTTON_ID = 'timerAlertButton'
+CONFIRMATION_ALERT_BUTTON_ID = 'confirmButton'
+PROMPT_ALERT_BUTTON_ID = 'promtButton'
+CONFIRMATION_RESULT_ID = 'confirmResult'
+PROMPT_RESULT_ID = 'promptResult'
+
+
+class TestToolsQAAlerts(BaseTest):
+    def setUp(self):
+        self._setUp(BASE_URL)
```

Comment:
Parent's class method should be called setUp() and you can call it like this:

super(TestToolsQAAlerts, self).setUp(URL)